inference-engine/tools/vpu/vpu_compile/main.cpp

```diff
@@ -105,6 +105,10 @@ static bool parseCommandLine(int *argc, char ***argv) {
 static std::map<std::string, std::string> configure(const std::string &configFile, const std::string &xmlFileName) {
     auto config = parseConfig(configFile);
 
+    IE_SUPPRESS_DEPRECATED_START
+        config[VPU_MYRIAD_CONFIG_KEY(PLATFORM)] = "VPU_MYRIAD_2480";
```

Comment:
For the problem we are now solving, it looks like a crutch. Anyway, this config key will be removed further. I think we should notify a plugin somehow that there is no problem and it can compile a network without a device.
So far I do not see a clear solution, but I can proposal
* Add new config key aka MYRIAD_MOCK_DEVICE = CONFIG_VALUE(YES/NO)
* Or create mock device even if config.platform() == NC_ANY_PLATFORM

Reply:
It needs to be discussed